src/gui/menu_items_open.cpp

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file menu_items_open.cpp
+ */
+
+#include "menu_items_open.hpp"
+#include "ScreenHandler.hpp"
+#include "ScreenFactory.hpp"
+#include "png_resources.hpp"
+#include "screen_sysinf.hpp"
+
+#include "screen_messages.hpp"
+#include "screen_menu_filament.hpp"
+#include "screen_menu_temperature.hpp"
+#include "screen_menu_move.hpp"
+#include "screen_menu_sensor_info.hpp"
+#include "screen_menu_odometer.hpp"
+#include "screen_menu_version_info.hpp"
+#include "screen_menu_fw_update.hpp"
+#include "screen_menu_languages.hpp"
+#include "screen_menu_lan_settings.hpp"
+#include "screen_menu_hw_setup.hpp"
+#include "screen_menu_eeprom.hpp"
+#include "screen_menu_footer_settings.hpp"
+#include "screen_prusa_link.hpp"
+#include "screen_menu_connect.hpp"
+#include "screen_menu_experimental_settings.hpp"
+#include "screen_menu_network.hpp"
+#include "screen_menu_eeprom_diagnostics.hpp"
+#include "gui/test/screen_menu_test.hpp"
+
+#define GENERATE_OPENER_ITEM_DEFINITION__(NAME, SCREEN, ICON, VISIBILITY) \
```

Comment:
I am very strongly against creating obfuscation define macros likes these. It's not really adding to readability (i have to go and find out what the macro actually does), it's bloating global namespace and it's nigh impossible to debug easily. 
Not to mention extendability goes also completely out of the picture.
And as a side note, you're doing in headers what could be done in sources.